tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -428,6 +428,32 @@ public function adminHasAddedUserToGroupUsingTheGraphApi(
 		}
 	}
 
+	/**
+	 * @When /^the administrator creates a group "([^"]*)" using the GraphApi$/
```

Comment:
```suggestion
	 * @When /^the administrator creates a group "([^"]*)" using the GraphAPI$/
```
API: Application Programming Interface